cmake/pcl_options.cmake

```diff
@@ -64,3 +64,9 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 option(BUILD_tools "Useful PCL-based command line tools" ON)
 
 option(WITH_DOCS "Build doxygen documentation" OFF)
+
+# set index size
+set(PCL_SIZE_INDEX 32 CACHE STRING "Set index size. Available options are: 8 16 32 64")
```

Comment:
Please read the note in `types.h` regarding the default size. It can't be 32 for PCL 1.11 due to compatibility. It can be 32 for PCL 1.12 since the user can then override the size at compile time

Reply:
Would modifying `types.h` to be
```c++

#if PCL_MINOR_VERSION <= 11
// sizeof returns bytes, while we measure size by bits in the template
#undef PCL_INDEX_SIZE
#define PCL_INDEX_SIZE (sizeof(int) * 8)
#else
#ifndef PCL_INDEX_SIZE
#define PCL_INDEX_SIZE 32
#endif  // PCL_INDEX_SIZE
#endif  // PCL_MINOR_VERSION
```
be an option or should the changes  be made solely in the cmake configuration?

Reply:
You can modify anything as long as it keeps the behavior (and is sane) 😄